.github/workflows/test.yml

```diff
@@ -91,8 +91,9 @@ jobs:
           FIGMA_TOKEN: ${{ secrets.FIGMA_TOKEN }}
           STREAMLINE_SECRET: ${{ secrets.STREAMLINE_SECRET }}
           STREAMLINE_FAMILIES: ${{ secrets.STREAMLINE_FAMILIES }}
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
I think both of the jobs need the token. If the build doesn't have the token, it won't build the contributors and the links to their profile pages won't get checked later.